### PR TITLE
[lab-supervised-learning] Angela Gomez

### PR DESCRIPTION
**Pull request by angelagomezf**
Hola Adri!! En general el lab me ha ido bien salvo el challenge 6 😬 Te lo he dejado apuntando en el Jupyter, pero me he quedado bloqueada en el punto donde tenía que hacer el *model_fit* ...

Alt-text

### CODE REVIEW

**Comment by adrianacoca** [comment]
Buenos días Angela!

En el challenge 2, has eliminado las columnas `"URL_LENGTH", "NUMBER_SPECIAL_CHARACTERS", "CONTENT_LENGTH"` para eliminar la colinearidad, pero deberías haber eliminado aquellas columnas que tenían valores de correlación muy altos. En este caso serían: 
- APP_BYTES                      
- SOURCE_APP_PACKETS             
- REMOTE_APP_PACKETS             
- SOURCE_APP_BYTES               
- REMOTE_APP_BYTES               
- APP_PACKETS                    
- TCP_CONVERSATION_EXCHANGE
descarga (2)

Que paso en el Challenge 6, te bloqueaste? 😢  El proceso es igual que con otros modelos!
Veo que has dividido los datos con el `train_test_split`, y has importado Logistic Regression. Después de esto:
- Creas el modelo:  `website_LR_model = LogisticRegression()`
- Lo entrenas con los datos de train: `website_LR_model.fit(X_train,y_train)`
- Predices los valores de y_test y los asignas a una variable: `y_pred = website_LR_model.predict(X_test)`
- Revisas los datos con la confusion matrix: `confusion_matrix(y_test, y_pred)`
- Revisas los datos con el accuracy score: `accuracy_score(y_test, y_pred)`

Buen trabajo 🚀 

MachineL
